backend/src/db/factories.js

```diff
@@ -27,7 +27,7 @@ export const cleanDatabase = async (options = {}) => {
 Factory.define('category')
   .attr('id', uuid)
   .attr('icon', 'globe')
-  .attr('name', 'global-peace-nonviolence')
+  .attr('name', 'Global Peace & Nonviolence')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T13:27:45Z" title="Tuesday, February 18th 2020, 2:27:45 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -27,7 +27,7 @@ export const cleanDatabase = async (options = {}) => {
 Factory.define('category')
   .attr('id', uuid)
   .attr('icon', 'globe')
-  .attr('name', 'global-peace-nonviolence')
+  .attr('name', 'Global Peace & Nonviolence')
```


This `name` gets translated, @mattwr18

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-18T13:30:25Z" title="Tuesday, February 18th 2020, 2:30:25 pm +01:00">Feb 18, 2020</time>_

---


no, the name does not, it is the `slug` that gets translated, and because it is calling this twice, there is no key in `en.json` so it just appears as `category.name.null`. This is a separate issue that was introduced by the `rosie` refactor, but I realized it doesn't matter, cause I don't care what category the user creates the post with, so I just select the first button